examples/app.py

```diff
@@ -91,14 +93,21 @@
 app.register_blueprint(blueprint)
 
 search = InvenioSearch(app)
-search.register_mappings('records', 'data')
+
+if ES_VERSION[0] == 2:
+    search.register_mappings('records', 'es2_data')
+elif ES_VERSION[0] == 5:
+    search.register_mappings('records', 'es5_data')
+else:
+    raise Exception('This example supports only elasticsearch version '
+                    '2.x and 5.x')
 
 
 class ExampleSearch(RecordsSearch):
     """Example search class."""
 
     class Meta:
-        index = 'demo'
+        index = 'records'
```

Comment:
I would try to keep the name `demo` or `example` in order to avoid confusion with `invenio-marc21` where we already have `records`.